source/css/sass/_grid.scss

```diff
@@ -1,16 +1,26 @@
+@import "variables/color";
+@import "mixins/grid";
 @import "mixins/spacing";
 
+// TODO: Remove this wrapper and account for it
 .wrapper {
   @include wrapper();
 }
 
 .grid {
-  @supports (display: grid) {
-    display: grid;
+  @include grid-setup(map_get($grid, column-count), map_get($grid, column-gap), map_get($grid, max-width));
+  width: 100vw;
+
+  & > * {
+    @include grid-span-full();
+  }
+
+  & > .grid-item--span-all {
```

Comment:
`.grid__row--span-all`? Unusual to have a modifier without an element too...

Reply:
Remnants of an abortive approach. Will 🔥